apps/spark/src/main/java/com/linkedin/openhouse/jobs/spark/DataLayoutStrategyGeneratorSparkApp.java

```diff
@@ -74,7 +74,7 @@ private void runInnerPartitionScope(
     List<DataLayoutStrategy> strategies = strategiesGenerator.generatePartitionLevelStrategies();
     log.info("Generated {} strategies for table {}", strategies.size(), fqtn);
     StrategiesDao dao = StrategiesDaoTableProps.builder().spark(spark).build();
-    dao.savePartitionScope(fqtn, strategies);
```

Comment:
Is there a plan to save in alternate location as table specific storage is needed?

Reply:
No plan yet.